test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -484,3 +509,28 @@ func clusterConditionShowsHookFailed(cluster *clusterv1.Cluster, hookName string
 		strings.Contains(conditions.GetMessage(cluster, clusterv1.TopologyReconciledCondition), hookFailedMessage) &&
 		strings.Contains(conditions.GetMessage(cluster, clusterv1.TopologyReconciledCondition), hookName)
 }
+
+func backupAndDeleteCluster(ctx context.Context, proxy framework.ClusterProxy, namespace, clusterName, artifactFolder string) {
```

Comment:
```suggestion
func dumpAndDeleteCluster(ctx context.Context, proxy framework.ClusterProxy, namespace, clusterName, artifactFolder string) {
```
nit: up to you. Just an idea. Fits rather to how we are using dump. When I read backup I think about backup/restore stuff